Streetcode/Streetcode.BLL/Interfaces/Streetcode/TextContent/ITextService.cs

```diff
@@ -0,0 +1,6 @@
+namespace Streetcode.BLL.Interfaces.Streetcode.TextContent;
+
+public interface ITextService
```

Comment:
We don't need some of the repository interfaces;
You can just make its realization dependent on IRepositoryBase.